src/components/UI/SearchBar/index.tsx

```diff
@@ -23,6 +23,17 @@ export default function SearchBar() {
         placeholder={t('search')}
         id="search"
       />
+      <span className="clearSearchInputWrapper">
+        <span
```

Comment:
Can you replace this with a `button` tag? that makes it more accessible, `span` tags are not good to grab keyboard focus if navigating the app without a mouse

Reply:
Should be out in the next commit